codesearch.k8s.io/config.json

```diff
@@ -0,0 +1,190 @@
+{
+	"max-concurrent-indexers": 2,
+	"dbpath": "data",
+	"repos": {
```

Comment:
Question from before - can we make it pull from *all* repos?

Reply:
also, do we need to index kubernetes-sigs and incubator?

Reply:
adding a python script to fetch all the ones from kubernetes/*, once we get this working, we can pull other orgs